### DISCUSSION TITLE
Feel Free to talk about This Project

### DISCUSSION THREAD

**Discussion by Cyogenus** [Cyogenus/IPTV-MAC-STALKER-PLAYER#3] [General]
If you have any questions or you want to help. You can post here

**Comment by sh6gw54gsh-stack** [comment]
Hi

I am currently developing an application that needs to be integrated with the Stalker portal. I need your support to ensure proper configuration and communication, could you please provide the following details:

 1. The correct portal URL path (used for the handshake process and API calls), including the handshake endpoint.
 2. The required HTTP headers that must be included in the requests.

Your assistance will be greatly appreciated.

Thanks very much
